### ISSUE TITLE
During refine selection workflow, if the original selection is removed, the notification list gets removed

### ISSUE THREAD

**Issue by aparveen** [Esri/solutions-components#186]
During refine selection workflow, if the original selection is removed but still have selection from the Refine workflow process, msg "there is no selections to refine, Go to My lists to create list of features" shows up

Repro Steps:
1) Create a list by searching an address without any buffer.
2) Go to the Refine tab and add few features and remove the searched feature, in this state you will not have the original selection but you still have features added through refine workflow.
3) Notice the message that there is no selection to refine.
in this state user should still have the notification list and the newly added features to work with
Test App: https://sg-em-qaext.mapsqa.arcgis.com/apps/instant/notification/index.html?appid=3b3e80d082c04bf18489bbd848ff699c

![refineselectionissue](https://user-images.githubusercontent.com/10148401/220486646-0b11f79c-1dbd-47f9-9bb6-f4eadbab49f9.gif)

**Comment by jmhauck** [comment]
The refine selections are maintained much like the other selection sets. We previously showed the refine set in the selection set list and download lists. However, we decided to go away from that (would have to look back to notes to try and recall all the details...could do so if necessary). So in a few places we just basically ignore the refine selection set in regards to its display.

We could easily remove the code that ignores the refine set in the refine page and when you get to the state as outlined above we would still be able to interactively add/remove.

A couple of things will need to think through. What if anything shows in the main list and download pages? Should a user be able to "Refine" at any point regardless of having any standard selection sets? If they have no standard selection set as is the case above and they add a feature to refine and then remove that feature so the refine set is empty should they be allowed to continue refining? 

Should we consider not removing a standard selection set if all of its features have been removed from refine? In this case just show the list with 0 features. Right now if you create a selection set...then remove some features with refine...then reopen the selection set to edit (when it re-opens the same geometry that was originally used to create the selection set is used to re-apply the same original selection...this is not saved unless you click Done again...I thought this would be helpful if you got to some weird place with refine and actually just wanted to go back to the original selection but not sure how apparent that is)

We can talk more...just wanted to jot down a few notes while it was on my mind.
